charts/assemble-backstage/templates/assemble-config-secret.yaml

```diff
@@ -0,0 +1,89 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: assemble-config
+stringData:
+  app-config.yaml: |- 
+    app:
+      # Should be the same as backend.baseUrl when using the `app-backend` plugin.
+      #baseUrl: http://{{ .Release.Name }}:7007
+      baseUrl: {{ .Values.backstage.baseUrl }}
+
+    organization:
+      name: "{{ .Values.backstage.companyname }}"
+
+    backend:
+      # Note that the baseUrl should be the URL that the browser and other clients
+      # should use when communicating with the backend, i.e. it needs to be
+      # reachable not just from within the backend host, but from all of your
+      # callers. When its value is "http://localhost:7007", it's strictly private
+      # and can't be reached by others.
+      #baseUrl: http://{{ .Release.Name }}:7007
+      baseUrl: {{ .Values.backstage.baseUrl }}
+      listen:
+        port: 7007
+        # The following host directive binds to all IPv4 interfaces when its value
+        # is "0.0.0.0". This is the most permissive setting. The right value depends
+        # on your specific deployment. If you remove the host line entirely, the
+        # backend will bind on the interface that corresponds to the backend.baseUrl
+        # hostname.
+        host: 0.0.0.0
+      csp:
+        connect-src: ["'self'", 'http:', 'https:']
+        # Content-Security-Policy directives follow the Helmet format: https://helmetjs.github.io/#reference
+        # Default Helmet Content-Security-Policy values can be removed by setting the key to false
+      cors:
+        origin: http://{{ .Release.Name }}:3000
+        methods: [GET, POST, PUT, DELETE]
+        credentials: true
+
+      # config options: https://node-postgres.com/api/client
+      database:
+        client: pg
```

Comment:
In memory option?

Reply:
The values file can support this but current image does not contain the sqllite module. Perhaps a new story should be created if this is needed.